libsolidity/codegen/ABIFunctions.h

```diff
@@ -62,7 +62,7 @@ class ABIFunctions
 		m_evmVersion(_evmVersion),
 		m_revertStrings(_revertStrings),
 		m_functionCollector(_functionCollector),
-		m_utils(_evmVersion, m_revertStrings, m_functionCollector)
+		m_utils(_evmVersion, std::nullopt, m_revertStrings, m_functionCollector)
```

Comment:
This one is a bit dangerous because `ABIFunctions` is also used in IR generation so its copy of `YulUtilFunctions` may be too. If we hard-coded `nullopt` here, things may break if we add something new depending on EOF to `YulUtilFunctions` because it assumes it does receive the version.